main-site/lib/data.ts

```diff
@@ -0,0 +1,24 @@
+import { AboutSectionData } from './types';
+
+import { nanoid } from 'nanoid';
+
+export const aboutSectionData: AboutSectionData[] = [
+  {
+    id: nanoid(),
```

Comment:
I learned recently it's bad practice to use nanoid() for the react key prop as shown in nanoid docs: https://github.com/ai/nanoid#react. not sure if it's an issue here since i dont think nanoid() is called every render since we're in `/lib` but this is def a problem in other areas of the site

Reply:
Do you recommend randomly generating one online and then copy pasting it? Or just using the title as the key?

Reply:
I would just use the title